package.json

```diff
@@ -2,11 +2,12 @@
   "name": "virtual-jade-loader",
   "version": "0.2.2",
   "description": "webpack loader for jade -> virtual-dom/hyperscript",
-  "main": "index.js",
+  "main": "ilib/index.js",
   "engines": {
     "node": ">=4.2"
   },
   "scripts": {
+    "lint": "node_modules/eslint/bin/eslint.js ./lib",
```

Comment:
ah ok didn't realize that

On Tue, Jun 7, 2016 at 6:52 PM, tdumitrescu <notifications@github.com>
wrote:

> In package.json
> <https://github.com/tdumitrescu/virtual-jade-loader/pull/2#discussion_r66172914>
> :
>
> >    "engines": {
> >      "node": ">=4.2"
> >    },
> >    "scripts": {
> > +    "lint": "node_modules/eslint/bin/eslint.js ./lib",
>
> npm run lint should be looking in node_modules before anything else
> globally installed
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tdumitrescu/virtual-jade-loader/pull/2/files/3a55fec5efa7a09500c95660a8ad73858ef534b4#r66172914>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAh_UjJJ7HFzyIOzrjL0hvxyxSaXfOo3ks5qJgRVgaJpZM4IwWu_>
> .
>